### PR TITLE
Ethan development

### PR DESCRIPTION
**Pull request by erigb**
Week 2 complete! I have created the database for my server side to save comments. Also, if I am using Gson, should I get rid of the Json stuff I have in my code? I assume I should but just want to be sure.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/Comment.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/*
+*This class is in charge of each comment object.
+*/
+package com.google.sps.data;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Comment  {
```
Comment (dgalbraith33) [inline portfolio/src/main/java/com/google/sps/data/Comment.java:9]: From a design perspective I think it makes more sense to have a Comment contain an individual name, location and comment and to return a list of comments from the server. I don't think it makes sense to change now that this works as is, but just a thought to keep in mind for the future.

My reasoning is that it is a bit less likely to have weird errors introduced as the code evolves in the future where the lists somehow end up out of sync. And it is a bit clearer on the API side that each (name, location, comment) pair goes together in the JSON

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -14,19 +14,108 @@
...
+
+      DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+      datastore.put(taskEntity);
+
+      response.sendRedirect("/index.html");  //Change this when using actual clinet/server
+  }
+
+   /**
+   * Converts a ServerStats instance into a JSON string using manual String concatentation.
+   * This is not used anymore since I switched to Gson. Should I delete it?
```
Comment (dgalbraith33) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:91]: Yes we should delete this. The pros of using version control is that if you ever decide you need this function again in the future it's easy enough to go grab it out of the history and add it back.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -14,19 +14,108 @@
...
+      Entity taskEntity = new Entity("Comment");
+      taskEntity.setProperty("name", name);
+      taskEntity.setProperty("location", location);
+      taskEntity.setProperty("comment", comment);
+      taskEntity.setProperty("timestamp", timestamp);
+
+      DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+      datastore.put(taskEntity);
+
+      response.sendRedirect("/index.html");  //Change this when using actual clinet/server
```
[the remark this answers is not in the document]
Comment (erigb) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:86]: I do not think I need this comment anymore

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (erigb) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:91]: Sounds good

File: portfolio/src/main/java/com/google/sps/data/Comment.java (hunk 1 above)
Comment (erigb) [reply portfolio/src/main/java/com/google/sps/data/Comment.java:9]: Hmm yeah that is true. I could have just made an ArrayList of comments. For some reason that did  not cross my mind.

**Pull request merged**